pyproject.toml

```diff
@@ -9,14 +9,27 @@ build-backend = "setuptools.build_meta"
 legacy_tox_ini = """
 [tox]
 
-[testenv]
+[testenv:tests]
 changedir = tests
 deps =
     coverage[toml]
 
 commands =
     coverage run -m unittest discover -p "*.py"
     coverage xml
+
+[testenv:examples]
+changedir = examples
+allowlist_externals = /bin/bash
+
+deps =
+    jupyter
+    jupyter_contrib_nbextensions
+    matplotlib
+    tqdm
```

Comment:
why is tqdm required here?

Reply:
because the `FeatureSelection.ipynb` uses it. Given that it is a nice visual key for the user how long the computation take, and that I don't believe installing it takes a lot of time, I think we should keep it.

Reply:
That's fair =)

What do you think of moving `tqdm` and `matplotlib` to `examples/requirements.txt`, and install these requirements as part of the test commands? This way the tox config only contains what is required to run the tests, all others dependencies are specified elsewhere.

Reply:
done